chart/values.yaml

```diff
@@ -26,6 +26,14 @@ image:
   # -- ImagePullPolicy for our images
   pullPolicy: Always
 
+  # docker-secrets required to pull images if the container registry from image.Registry is protected
+  pullSecrets:
+    # -- Enable pullSecrets for pulling our container images
+    enabled: false
+    # -- Name of the pullSecret in the installed namespace
+    secrets:
+      - name: login
```

Comment:
This (secrets)name is not going to change in this repo I'd believe?

Reply:
Actually there is no default name any more in the latest change.